app/api.py

```diff
@@ -1,6 +1,7 @@
 import json
 import requests
 import urllib
+import logging

```

Comment:
imports should be sorted alphabetically.

Reply:
got it.